src/app/thumbnails.h

```diff
@@ -1,56 +1,23 @@
-// Aseprite
-// Copyright (C) 2016  Carlo Caputo
-//
-// This program is free software; you can redistribute it and/or modify
-// it under the terms of the GNU General Public License version 2 as
-// published by the Free Software Foundation.
-
```

Comment:
No need to remove the copyright notice (my comment about the `thumbnails.h` content was just illustrative 😄 )